en/orm/retrieving-data-and-resultsets.rst

```diff
@@ -371,6 +371,23 @@ The above would translate into the following::
     additional performance overhead.
 
 
+Retrieving Associated Data
+==========================
+
+When you want to grab associated data, or filter based on associated data, there are basically two ways:
+ - use CakePHP ORM query functions like ``contain()`` and ``matching()``
+ - use join functions like ``innerJoin()``, ``leftJoin()``, and ``rightJoin()``
+ 
+When do you use ``contain()``?
+You use it when the main model hasOne or belongsTo the associated data models. 
```

Comment:
Not entirely true. You can use `contain` on any association type. The difference is that contain will not filter the parent association based on the results of the child, whereas `matching` will. I think explaining the difference based on "filtering" is important